js/cli/src/run.ts

```diff
@@ -42,8 +43,47 @@ export const run = async (options: RunOptions): Promise<void> => {
     "--build-tool-version",
     versions.gatling.jsAdapter
   ];
+  return run(options, "io.gatling.app.Gatling", additionalClasspathElements, javaArgs, simulationArgs);
+};
+
+export const runRecorder = async (options: RunRecorderOptions): Promise<void> => {
+  logger.info("Running the Gatling Recorder");
+
+  const recorderArgs = [
+    "--simulations-folder",
+    options.sourcesFolder,
+    "--resources-folder",
+    options.resourcesFolder,
+    "--format",
+    options.typescript ? "typescript" : "javascript"
+  ];
+  return run(options, "io.gatling.recorder.GatlingRecorder", [], [], recorderArgs);
+};
+
+const run = (
+  options: RunOptions,
+  mainClass: string,
+  additionalClasspathElements: string[],
+  javaArgs: string[],
+  mainClassArgs: string[]
+): Promise<void> => {
+  const command = `${options.graalvmHome}/bin/java`;
+  const classpathSeparator = osType === "Windows_NT" ? ";" : ":";
+  const classpath = [...additionalClasspathElements, options.jvmClasspath].join(classpathSeparator);
```

Comment:
I'm concerned about this. The command length is limited on Windows and we can easily reach the limit because of the depth hierarchy in maven/ivy local caches. That's why we have the special launcher in the plugins. We might want to use that.

Reply:
As discussed: added to our TODO-list on Notion, we'll see later if it's needed in some cases.